src/govuk/components/character-count/character-count.js

```diff
@@ -6,7 +6,7 @@ function CharacterCount ($module) {
   this.$module = $module
   this.$textarea = $module.querySelector('.govuk-js-character-count')
   if (this.$textarea) {
-    this.$countMessage = document.getElementById(this.$textarea.id + '-info')
```

Comment:
Is it worth adding a rudimentary `this.$countMessage = null` here? This would primarily be a readability thing as otherwise the declaration is hidden on line 39 which might be a hassle for devs to find if they're trying to trace how this module works. Not a blocker.